src/hooks/tradingView/useTradingViewTheme.ts

```diff
@@ -89,13 +88,15 @@ export const useTradingViewTheme = ({
           }
 
           // Necessary to update existing chart lines
-          Object.entries(orderLines).forEach(([key, line]) => {
-            const { orderColor, borderColor, backgroundColor, textColor, textButtonColor } =
-              getOrderLineColors({ side: key.split('-')[0], appTheme, appColorMode });
+          Object.entries(chartLines).forEach(([key, line]) => {
+            const { maybeQuantityColor, borderColor, backgroundColor, textColor, textButtonColor } =
+              getChartLineColors({ chartLineType: key.split('-')[0], appTheme, appColorMode });
```

Comment:
`chartLineType` is not expecting a `string`. I see why you added the `side.rawValue` to the key now. Is there a better way to pass `side` to this fn?

Reply:
ah yeah - updated the `ChartLine` type to hold this info!